www/src/components/repository/queries.js

```diff
@@ -18,6 +18,18 @@ export const CREATE_REPOSITORY_MUTATION = gql`
   ${RepoFragment}
 `
 
+export const UPDATE_REPOSITORY_MUTATION = gql`
+  mutation UpdateRepository($repositoryId: ID!, $attributes: RepositoryAttributes!) {
+    updateRepository(repositoryId: $repositoryId, attributes: $attributes) {
+      ...RepoFragment
+      tags {
+        name: tag
```

Comment:
It might be best to not do this rename so the cache is consistent elsewhere that doesn't do that